src/databases/functions/users/set-w-pu-user.js

```diff
@@ -0,0 +1,13 @@
+const {User} = require('../../models');
+
+module.exports = async (userId, score) => {
+  if (!userId || !score) return null;
+
+  try {
+    const result = await User.update({w_pu_user: score}, {where: {user_id: userId}});
+    return result;
+  } catch (error) {
+    console.error(`Failed to update user w pu with id ${userId}: `, error);
+    return null;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Avoid using `console.error` for error handling in production code. It's better to use a logging library that provides more control over log levels, formats, and destinations. Also, consider throwing the error after logging it so that the caller of this function knows something went wrong.
```diff
-    console.error(`Failed to update user w pu with id ${userId}: `, error);
-    return null;
+    logger.error(`Failed to update user w pu with id ${userId}: `, error);
+    throw error;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->